src/taco-utils/tacoError.ts

```diff
@@ -76,7 +76,7 @@ module TacoUtility {
                     message = resources.getString(errorToken, args);
                 }
             } else {
```

Comment:
I thought about that at first, but then I saw that `args` is an array, not a `...args: any[]`. So, I'm afraid that if I pass `args` directly to `getString()`, it will treat the array as the first argument rather than as a list of several arguments. If that's the case, it means we would need to do some massaging to invoke `getString()` with each element of the array as its own separate argument.

If there is an easy way to just pass an array to a `...args: any[]` type, then we could remove this method entirely and use `resources.getString()` directly in the callers